schema/operating_rules.json

```diff
@@ -0,0 +1,113 @@
+      "type": "integer",
+      "minimum": 1450155600
+    },
+    "ttl": {
+      "description": "Number of seconds before the data in the feed will be updated again (0 if the data should always be refreshed).",
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description": "GOFS-lite version number to which the feed conforms, according to the versioning framework (added in v1.1).",
+      "type": "string",
+      "enum": [
+        "1.0"
+      ]
+    },
+    "data": {
+      "type": "object",
+      "required": [
+        "operating_rules"
+      ],
+      "properties": {
+        "operating_rules": {
+          "type": "array",
+          "description": "Array that contains one object per operating rule as defined below.",
+          "items": {
+            "type": "object",
+            "title": "A Schema",
+            "required": [
+              "from_zone_id",
+              "to_zone_id",
+              "calendars",
+              "vehicle_type_id"
+            ],
+            "properties": {
+              "from_zone_id": {
+                "type": "string",
+                "title": "ID from a zone defined in zones.json representing the boarding zone for the current rule."
+              },
+              "to_zone_id": {
+                "type": "string",
+                "title": "ID from a zone defined in zones.json representing the alighting zone for the current rule. from_zone_id and to_zone_id MAY reference the same zone."
+              },
+              "start_pickup_window": {
+                "type": "string",
+                "title": "Time at which the pickup starts being available in from_zone_id defined in this array. If start_pickup_window is provided, either end_pickup_window or end_dropoff_window MUST also be provided."
+              },
+              "end_pickup_window": {
+                "type": "string",
+                "title": "Time at which the pickup stops being available in from_zone_id defined in this array. If end_pickup_window is provided, start_pickup_window MUST be provided."
+              },
+              "calendars": {
+                "type": "array",
+                "title": "Array of calendar IDs from calendars.json defining the dates and days when the pickup and drop off occur.",
+                "items": {
+                  "type": "string",
+                  "title": "A Schema",
+                  "examples": [
+                    "all-days"
+                  ]
+                }
+              },
+              "end_dropoff_window": {
+                "type": "string"
+              },
+              "brand_id": {
+                "type": "string"
+              },
+              "vehicle_type_id": {
+                "description": "Array of vehicle types used for delivering the on-demand service.",
+                "type": "array"
+              },
+              "fare_id": {
+                "type": "string"
+              }
```

Comment:
I'm testing with some real data and the results are failing because it's got the optional fare ID field as`"fare_id": null`. The kind of small error we'd likely never catch unless/until it broke something.
I point it out not so much because I think you need to change anything in this PR, but rather just to highlight how useful of a tool this schema will be.